guides/v2.1/cloud/env/variables-deploy.md

```diff
@@ -424,15 +424,29 @@ stage:
 -  **Default**—`true`
 -  **Version**—Magento 2.1.4 and later
 
-On deployment, replace Magento base URLs in the database with project URLs. This is useful for local development, where base URLs are set up for your local environment. When you deploy to a Cloud environment, we change the URLs so you can access your storefront and Magento Admin using project URLs.
+On deployment, replace Magento base URLs in the database the project URLs specified in the `MAGENTO_CLOUD_ROUTES` variable. This is useful for local development, where base URLs are set up for your local environment. 
```

Comment:
` replace Magento base URLs in the database the project URLs specified in the`
I'm not sure but maybe `with` is missed.
` replace Magento base URLs in the database with the project URLs specified in the`